example/gardener-local/kind/.gitignore

```diff
@@ -1 +1,2 @@
 kubeconfig
+kubeconfig2
```

Comment:
This should not be needed anymore when the kubeconfig for the second kind cluster is stored in `example/gardener-local/kind2`. You probably need to copy this `.gitignore` file to the `kind2` folder.